internal/kibana/reconciler.go

```diff
@@ -567,6 +567,7 @@ func newKibanaPodSpec(cluster *KibanaRequest, elasticsearchName string, proxyCon
 		"-cookie-secret-file=/secret/session-secret",
 		"-cookie-expire=24h",
 		"-skip-provider-button",
+		"-skip-auth-regex=^/api/status",
```

Comment:
If there are no subpaths in `/api/status`, I'd limit the regex to just that URL (assuming the regex-parser understands this, but if it knows ^ it should also know $):

```suggestion
		"-skip-auth-regex=^/api/status$",
```